win32 port: undefined reference to mstimer_now'

```
root@debian:/home/kmd/bacnet-stack# make  BACNET_PORT=win32  router-mstp
make -s -C apps router-mstp
/usr/bin/i686-w64-mingw32-ld: /home/kmd/bacnet-stack/src/bacnet/basic/sys/mstimer.o:mstimer.c:(.text$mstimer_set+0xf): undefined reference to `mstimer_now'
/usr/bin/i686-w64-mingw32-ld: /home/kmd/bacnet-stack/src/bacnet/basic/sys/mstimer.o:mstimer.c:(.text$mstimer_elapsed+0x7): undefined reference to `mstimer_now'
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:63: router-mstp.exe] Error 1
make[1]: *** [Makefile:287: router-mstp] Error 2
make: *** [Makefile:110: router-mstp] Error 2




```

--- a/apps/router-ipv6/main.c
+++ b/apps/router-ipv6/main.c
@@ -56,8 +56,6 @@
 #include "bacnet/datalink/bvlc6.h"
 #include "bacnet/datalink/bip6.h"
 #include "bacnet/basic/bbmd6/h_bbmd6.h"
-#undef MAX_HEADER
-#undef MAX_MPDU
 #include "bacnet/datalink/bip.h"
 #include "bacnet/datalink/bvlc.h"
 #include "bacnet/basic/bbmd/h_bbmd.h"

--- a/apps/router-mstp/main.c
+++ b/apps/router-mstp/main.c
@@ -54,8 +54,6 @@
 #include "bacport.h"
 /* our datalink layers */
 #include "bacnet/datalink/dlmstp.h"
-#undef MAX_HEADER
-#undef MAX_MPDU
 #include "bacnet/datalink/bip.h"
 #include "bacnet/datalink/bvlc.h"
 #include "bacnet/basic/bbmd/h_bbmd.h"

--- a/ports/win32/dlmstp-mm.c
+++ b/ports/win32/dlmstp-mm.c
@@ -126,7 +126,7 @@ int dlmstp_send_pdu(BACNET_ADDRESS *dest, /* destination address */
             Transmit_Packet.pdu[i] = pdu[i];
         }
         bacnet_address_copy(&Transmit_Packet.address, dest);
-        bytes_sent = pdu_len + MAX_HEADER;
+        bytes_sent = pdu_len + DLMSTP_HEADER_MAX;
         Transmit_Packet.ready = true;
     }
 
@@ -282,7 +282,7 @@ uint16_t MSTP_Get_Send(
     } else {
         destination = MSTP_BROADCAST_ADDRESS;
     }
-    if ((MAX_HEADER + Transmit_Packet.pdu_len) > DLMSTP_MPDU_MAX) {
+    if ((DLMSTP_HEADER_MAX + Transmit_Packet.pdu_len) > DLMSTP_MPDU_MAX) {
         return 0;
     }
     /* convert the PDU into the MSTP Frame */
@@ -421,7 +421,7 @@ uint16_t MSTP_Get_Reply(
     } else {
         return 0;
     }
-    if ((MAX_HEADER + Transmit_Packet.pdu_len) > DLMSTP_MPDU_MAX) {
+    if ((DLMSTP_HEADER_MAX + Transmit_Packet.pdu_len) > DLMSTP_MPDU_MAX) {
         return 0;
     }
     /* is this the reply to the DER? */

--- a/ports/win32/dlmstp.c
+++ b/ports/win32/dlmstp.c
@@ -111,7 +111,7 @@ int dlmstp_send_pdu(BACNET_ADDRESS *dest, /* destination address */
             Transmit_Packet.pdu[i] = pdu[i];
         }
         bacnet_address_copy(&Transmit_Packet.address, dest);
-        bytes_sent = pdu_len + MAX_HEADER;
+        bytes_sent = pdu_len + DLMSTP_HEADER_MAX;
         Transmit_Packet.ready = true;
     }
 
@@ -267,7 +267,7 @@ uint16_t MSTP_Get_Send(
     } else {
         destination = MSTP_BROADCAST_ADDRESS;
     }
-    if ((MAX_HEADER + Transmit_Packet.pdu_len) > DLMSTP_MPDU_MAX) {
+    if ((DLMSTP_HEADER_MAX + Transmit_Packet.pdu_len) > DLMSTP_MPDU_MAX) {
         return 0;
     }
     /* convert the PDU into the MSTP Frame */
@@ -407,7 +407,7 @@ uint16_t MSTP_Get_Reply(
     } else {
         return 0;
     }
-    if ((MAX_HEADER + Transmit_Packet.pdu_len) > DLMSTP_MPDU_MAX) {
+    if ((DLMSTP_HEADER_MAX + Transmit_Packet.pdu_len) > DLMSTP_MPDU_MAX) {
         return 0;
     }
     /* is this the reply to the DER? */

